login.css

```diff
@@ -48,7 +48,7 @@ nav a{
 	padding: 20px 15px;
 	text-decoration:none;
 	font-family:Arial;
-	color:black;
+	color:white;
```

Comment:
06/08/2021: I make the color font for the link in the navbar to be white because it white and navy blue was a good combination that will work well together.